intro_to_ML/decisiontreemodel.ipynb

```diff
@@ -0,0 +1,285 @@
+       "mean   4.766729e+06   5150.541284    2.965138    1.286239    1.805505   \n",
+       "std    1.870440e+06   2170.141023    0.738064    0.502470    0.867492   \n",
+       "min    1.750000e+06   1650.000000    1.000000    1.000000    1.000000   \n",
+       "25%    3.430000e+06   3600.000000    2.000000    1.000000    1.000000   \n",
+       "50%    4.340000e+06   4600.000000    3.000000    1.000000    2.000000   \n",
+       "75%    5.740000e+06   6360.000000    3.000000    2.000000    2.000000   \n",
+       "max    1.330000e+07  16200.000000    6.000000    4.000000    4.000000   \n",
+       "\n",
+       "          parking  \n",
+       "count  545.000000  \n",
+       "mean     0.693578  \n",
+       "std      0.861586  \n",
+       "min      0.000000  \n",
+       "25%      0.000000  \n",
+       "50%      0.000000  \n",
+       "75%      1.000000  \n",
+       "max      3.000000  "
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# filepath \n",
+    "housing = 'Housing.csv'\n",
+    "# read the data and store data in DataFrame titled housing_data\n",
+    "housing_data = pd.read_csv(housing) \n",
+    "# print a summary of the data in housing data\n",
+    "housing_data.describe()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "y = housing_data.price"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "house_features = ['area', 'bedrooms', 'bathrooms', 'stories', 'parking']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "X = housing_data[house_features]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "1127424.0875912409\n"
+     ]
+    }
+   ],
+   "source": [
+    "from sklearn.tree import DecisionTreeRegressor\n",
+    "from sklearn.model_selection import train_test_split\n",
+    "from sklearn.metrics import mean_absolute_error\n",
+    "\n",
+    "# split data into training and validation data, for both features and target\n",
+    "# The split is based on a random number generator. Supplying a numeric value to\n",
+    "# the random_state argument guarantees we get the same split every time we\n",
+    "# run this script.\n",
+    "train_X, val_X, train_y, val_y = train_test_split(X, y, random_state = 0)\n",
+    "# Define model\n",
+    "house_model = DecisionTreeRegressor()\n",
+    "# Fit model\n",
+    "house_model.fit(train_X, train_y)\n",
+    "\n",
+    "# get predicted house prices on validation data\n",
+    "val_predictions = house_model.predict(val_X)\n",
+    "print(mean_absolute_error(val_y, val_predictions))"
```

Comment:
You could add a string describing what is being printed here as part of the print statement so it is clear to viewers of this doc who are not also reading the blog post.